README.adoc

```diff
@@ -1,43 +1,143 @@
-[#apicurio-registry-operator]
 = Apicurio Registry Operator
 
-[#requirements]
-== Requirements
+Apicurio Registry Operator provides a quick and easy way to deploy and manage
+https://github.com/Apicurio/apicurio-registry[Apicurio Registry] on Kubernetes or OpenShift.
+
+== Deployment Requirements
+
+The Apicurio Registry Operator supports the following platform versions:
+
+.Apicurio Registry Operator platforms
+[%header,cols=2*]
+|===
+| Platform | Required version
+| Kubernetes
+| 1.19+
+| OpenShift
+| 4.6+
+|===
+
+== Build Requirements
 
 * Docker
-* https://github.com/golang/go[go] (1.14+, with `export GO111MODULE='on'`), and `$GOPATH` and `$GOROOT` set.
-* https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md[Operator SDK] v1.4+
-* A running Kubernetes, https://kubernetes.io/docs/tasks/tools/install-minikube/[Minikube], OpenShift, or Minishift deployment with system admin access.
+* https://golang.org/[go] v1.14+ (with `export GO111MODULE='on'`, `$GOPATH` and `$GOROOT`)
+* https://sdk.operatorframework.io/docs/installation/[Operator SDK] v1.4+
+* A running Kubernetes (https://minikube.sigs.k8s.io/docs/start/[Minikube]) or OpenShift (https://www.okd.io/minishift/[Minishift]) cluster with system admin access
 
-[#quickstart]
 == Quickstart
 
-1. Prepare your Kubernetes cluster.
-If you are using Minikube, run `minikube addons enable ingress`.
+This section explains how to quickly install *development version* of the Apicurio Registry Operator on the command line.
+
+NOTE: The recommended installation option for non-development environments is https://operatorhub.io/operator/apicurio-registry[OperatorHub.io].
+The recommended storage options are SQL or Kafka.
+Please visit the latest https://www.apicur.io/registry/docs[Apicurio Registry and Operator documentation].
+
+.Procedure
+. Prepare your Kubernetes or OpenShift cluster.
+If you are using Minikube, run:
++
+[source,bash]
+----
+minikube addons enable ingress
+----
+
+. Build the latest distribution bundle:
++
+[source,bash]
+----
+make dist
+----
++
+NOTE: Run `make` without any parameters to get a help message.
+
+. Follow the instructions in `./dist/README.adoc` file.
+
+== Build and Deploy Apicurio Registry Operator
+
+This section explains how to build Apicurio Registry Operator on your own and deploy it for testing and development.
+
+.Procedure
+. Prepare your Kubernetes or OpenShift cluster.
+If you are using Minikube, run:
++
+[source,bash]
+----
+minikube addons enable ingress
+----
++
+NOTE: Run `export CLIENT="oc"` if you are deploying to OpenShift.
+
+. Choose a namespace where the operator will be deployed:
```

Comment:
Suggest uppercase O for Operator in all cases to match Kubernetes standards. Occurs in a few places in this readme.